client-python/pycti/api/opencti_api_client.py

```diff
@@ -91,9 +91,13 @@
 _PROXY_SIGNAL_HANDLERS_REGISTERED = False
 
 
-def build_request_headers(token: str, custom_headers: str, app_logger):
+def build_request_headers(token: str, custom_headers: str, app_logger, provider: str):
```

Comment:
we should rather cumulate the information in the user agent. eg
* no additional parameter -> `pycti/6.8.15` (as today)
* with additional parameter `client/1.2.3` ->   `pycti/6.8.15 client/1.2.3`